dali/test/python/test_backend_impl_torch_dlpack.py

```diff
@@ -63,7 +63,7 @@ def test_dlpack_tensor_list_gpu_to_cpu():
 
 
 def check_dlpack_types_gpu(t):
-    arr = torch.tensor([[-0.39, 1.5], [-1.5, 0.33]], device="cuda", dtype=t)
+    arr = torch.tensor([[0.39, 1.5], [1.5, 0.33]], device="cuda", dtype=t)
```

Comment:
The latest torch complains `value cannot be converted to type uint8 without overflow`.